spec/spec_helper.rb

```diff
@@ -6,25 +6,12 @@
 # Run Coverage report
 require 'solidus_dev_support/rspec/coverage'
 
-require 'rails-controller-testing'
-require 'rspec/active_model/mocks'
-
 # Create the dummy app if it's still missing.
 dummy_env = "#{__dir__}/dummy/config/environment.rb"
 system 'bin/rake extension:test_app' unless File.exist? dummy_env
 require dummy_env
 
-# Requires factories and other useful helpers defined in spree_core.
-require 'solidus_dev_support/rspec/feature_helper'
-require 'spree/testing_support/caching'
-require 'spree/testing_support/order_walkthrough'
-require 'spree/testing_support/translations' unless Spree.solidus_gem_version < Gem::Version.new('2.11')
-
-# Requires supporting ruby files with custom matchers and macros, etc,
-# in spec/support/ and its subdirectories.
-Dir["#{__dir__}/support/**/*.rb"].sort.each { |f| require f }
-
-SolidusDevSupport::TestingSupport::Factories.load_for(SolidusStarterFrontend::Engine)
+require 'capybara'
 
 Capybara.register_driver :apparition_docker_friendly do |app|
```

Comment:
Could we move the registration of the driver to `spec/support/solidus_starter_frontend/capybara.rb`? We need to make the tests pass when executed through Docker from within the `sandbox` application. I know this is something related to the development of the gem in contrast to the host application, but the same can be said of the other registered driver (`:apparition`), so I guess we don't have a better option :man_shrugging:

Reply:
Okay, I'll move the driver to `spec/support/solidus_starter_frontend/capybara.rb`. Todo.